wolfssl/wolfcrypt/rsa.h

```diff
@@ -434,11 +434,11 @@ WOLFSSL_API int wc_RsaExportKey(RsaKey* key,
                                           int nlen, int* isPrime);
 #endif
 
-WOLFSSL_LOCAL int wc_RsaPad_ex(const byte* input, word32 inputLen, byte* pkcsBlock,
+MP_API int wc_RsaPad_ex(const byte* input, word32 inputLen, byte* pkcsBlock,
```

Comment:
I thought MP was for multi precision and related to the math library API's. Why was MP_API chosen here?

Reply:
MP_API allows this function to be public (non-opaque / WOLFSSL_API) when `-DWOLFSSL_PUBLIC_MP` is defined. Otherwise MP_API defaults to opaque (WOLFSSL_LOCAL). MP_API was selected as the "configurable" visibility option.

Reply:
From wolfmath.h:

```
 36 #ifdef WOLFSSL_PUBLIC_MP                                                         
 37     #define MP_API   WOLFSSL_API                                                 
 38 #else                                                                            
 39     #define MP_API   WOLFSSL_LOCAL                                               
 40 #endif 
```